subsys/canbus/isotp/isotp_internal.h

```diff
@@ -54,9 +54,9 @@
 
 #define ISOTP_4BIT_SF_MAX_CAN_DL   8
 
-#define ISOTP_BS (CONFIG_ISOTP_BS_TIMEOUT)
-#define ISOTP_A  (CONFIG_ISOTP_A_TIMEOUT)
-#define ISOTP_CR (CONFIG_ISOTP_CR_TIMEOUT)
+#define ISOTP_BS_TIMEOUT (CONFIG_ISOTP_BS_TIMEOUT)
+#define ISOTP_A_TIMEOUT  (CONFIG_ISOTP_A_TIMEOUT)
+#define ISOTP_CR_TIMEOUT (CONFIG_ISOTP_CR_TIMEOUT)
```

Comment:
Since these are in milliseconds, how about: 

```suggestion
#define ISOTP_BS_TIMEOUT_MS (CONFIG_ISOTP_BS_TIMEOUT)
#define ISOTP_A_TIMEOUT_MS  (CONFIG_ISOTP_A_TIMEOUT)
#define ISOTP_CR_TIMEOUT_MS (CONFIG_ISOTP_CR_TIMEOUT)
```